mmv1/third_party/terraform/tests/universe_template_test.go

```diff
@@ -0,0 +1,46 @@
+package google
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+)
+func TestAccUniverseDomain(t *testing.T) {
+	t.Parallel()
```

Comment:
Note that we should skip or fail the test if key values are unset, with clear errors messages. That will mean adding a handler function for `GOOGLE_UNIVERSE_DOMAIN` (or a similar env var) and a handler (either a function or inline) for a custom domain until those are no longer required.

Reply:
Sure, I will leave this test skipped.